Rendera/include/scene.hpp

```diff
@@ -16,6 +16,7 @@ class Hitinfo{
         color shade_new = {0,0,0};
         color rgb;
         ray ref_ray;
+    Hitinfo() = default;
```

Comment:
Again, this line is not needed because the compiler will auto-generate it for you.